Source/TestDafny/MultiBackendTest.cs

```diff
@@ -94,10 +96,10 @@ record ResolutionSetting(
     int ExpectedExitCode
   );
 
-  private int ForEachCompiler(ForEachCompilerOptions options) {
-    var parseResult = CommandRegistry.Create(TextWriter.Null, TextWriter.Null, TextReader.Null,
-      new string[] { "verify", options.TestFile! }.Concat(options.OtherArgs).ToArray());
-    var dafnyOptions = ((ParseArgumentSuccess)parseResult).DafnyOptions;
+  private async Task<int> ForEachCompiler(ForEachCompilerOptions options) {
```

Comment:
Out of curiousity why all the new `async` introductions?

Reply:
This called `Task<int> DafnyCli.Execute(..)` before, so it needed to, but even without that call this should be async style. Right now it's 50% using async which is confusing, but I have another PR that takes it all the way there, so I wouldn't worry too much about it.